### PR TITLE
replaced init command with a simpler implementation

### PR DESCRIPTION
**Pull request by aricart**
new implementation only takes two options, 'name' and 'url', and creates an operator, account, and user using the specified name. If url is provided, it also deploys the created account to the specified url.

### CODE REVIEW

File: cmd/init.go [hunk 1]
```diff
@@ -19,304 +19,243 @@ import (
...
-			if p.operator.create {
-				cmd.Println("Success! - initialized environment directory")
+			if params.Deploy == "" {
+				cmd.Printf("\n\nTo run a local server using this configuration, enter:\n")
+				cmd.Printf("> nsc generate config --mem-resolver --config-file <path/server.conf>\n")
+				cmd.Printf("start a nats-server using the generated config:\n")
+				cmd.Printf("> nats-server -c <path/server.conf>\n\n")
+
+				cmd.Printf("Or deploy your account to a managed service enter:\n")
+				cmd.Printf("> nsc deploy --url https://jwt.ngs.local:6060/jwt/v1/operator\n\n")
```
Comment (sasbury) [inline cmd/init.go]: worried about putting a real URL here, maybe just <operator url>

File: cmd/init.go [hunk 2]
```diff
@@ -19,304 +19,243 @@ import (
...
-		p.operator.name = fmt.Sprintf("%s_operator", p.environmentName)
+func (p *EasyCmdParams) setOperatorDefaults(ctx ActionCtx) error {
+	oc, err := ctx.StoreCtx().Store.ReadOperatorClaim()
+	if err != nil {
+		return err
 	}
-
-	if p.account.name == "" {
-		p.account.name = fmt.Sprintf("%s_account", p.environmentName)
+	oc.OperatorServiceURLs.Add("nats://localhost:4222")
```
Comment (sasbury) [inline cmd/init.go]: should we tell them we did this somwehere?

File: cmd/init.go [hunk 3]
```diff
@@ -19,304 +19,243 @@ import (
...
+				cmd.Printf("> nsc generate config --mem-resolver --config-file <path/server.conf>\n")
+				cmd.Printf("start a nats-server using the generated config:\n")
+				cmd.Printf("> nats-server -c <path/server.conf>\n\n")
+
+				cmd.Printf("Or deploy your account to a managed service enter:\n")
+				cmd.Printf("> nsc deploy --url https://jwt.ngs.local:6060/jwt/v1/operator\n\n")
 			}
 
+			cmd.Printf("To listen for messages enter:\n")
+			cmd.Printf("> nsc tools sub \">\"\n")
```
Comment (sasbury) [inline cmd/init.go]: maybe this is where we can say the operator JWT defaults to pointing at a local server

File: cmd/init.go (hunk 2 above)
Comment (aricart) [reply cmd/init.go]: this is just a default value so that the:

nsc generate config ... 
stuff allows them to use the tools

Comment (aricart) [reply cmd/init.go]: they need to change it to their env whatever that is.

**Pull request merged**